demisto_sdk/commands/content_graph/objects/content_item.py

```diff
@@ -233,6 +236,8 @@ def dump(
         dir: DirectoryPath,
         marketplace: MarketplaceVersions,
     ) -> None:
+        if not self.path.exists():
+            logger.warning(f"Could not find file {self.path}, skipping dump")
```

Comment:
wouldn't it be raised in the exception below anyway?

Reply:
If this is the case we will skip dumping and not call `self.data`.